test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/CheckAllAnswersAndSubmitControllerSpec.scala

```diff
@@ -161,6 +164,18 @@ class CheckAllAnswersAndSubmitControllerSpec
       .expects(cgtReference, chargeReference, amount, returnUrl, backUrl, *, *)
       .returning(EitherT.fromEither[Future](response))
 
+  val mockRepresenteeCgtReference = RepresenteeCgtReference(CgtReference("12345"))
+
+  def mockCgtRegistrationService() =
```

Comment:
for more complete coverage you could expect what inputs you're expecting and also control the response to be able to test when things go wrong